src/main/kotlin/com/angorasix/projects/presentation/presentation/handler/ProjectsPresentationHandler.kt

```diff
@@ -129,7 +129,11 @@ class ProjectsPresentationHandler(
                 "Project Presentation",
             )
         }
-        return service.updateProjectPresentation(projectId, updateProjectPresentationData)?.let {
+        return service.updateProjectPresentation(
```

Comment:
Let's add a type check like we have in the `createProjectPresentation` method of this class:
```
return if (requestingContributor is SimpleContributor) {
...
} else {
    resolveBadRequest("Invalid Contributor Token", "Contributor Token")
}
```
(Note: we were indicating "Contributor **Header** in that method, but we're no longer using a Header for the Contributor data)

Also,  with the check we can then avoid casting with `as SimpleContributor` (when invoking this and the `convertToDto` methods)  at all, Kotlin realizes we have already checked the type of the object 👍